guide/C/ch_reports.xml

```diff
@@ -728,65 +728,89 @@ an empty report will be generated.</para>
     <para>The following use cases will be illustrated, and the options to be selected explained:</para>
-    <qandaset>
+    <qandaset defaultlabel='none'>
       <qandaentry>
         <question>
           <para>How much did I spend on various expense accounts last year?</para>
         </question>
         <answer>
-          <para>Accounts: click Expense, and Select Children</para>
-          <para>General: Choose relative dates “Start of Previous Year” and “End of Previous Year”</para>
-          <para>Sorting: Primary Key: Accounts, Secondary Key: Date</para>
-          <para>Sorting: Primary Key Subtotal: Enabled, Secondary Key Subtotal: None </para>
+          <para>The following sequence will select all subaccounts from the parent Expense account,
+          select appropriate dates, and sort by account and date. This will usually be the most
+          basic transaction report format.
+          </para>
+          <orderedlist>
+            <listitem><para><emphasis>Accounts</emphasis>: click Expense, and Select Children</para></listitem>
+            <listitem><para><emphasis>General</emphasis>: Choose relative dates “Start of Previous Year” and “End of Previous Year”</para></listitem>
+            <listitem><para><emphasis>Sorting</emphasis>: Primary Key: Accounts, Secondary Key: Date</para></listitem>
+            <listitem><para><emphasis>Sorting</emphasis>: Primary Key Subtotal: Enabled, Secondary Key Subtotal: None </para></listitem>
+          </orderedlist>
         </answer>
       </qandaentry>
       <qandaentry>
         <question>
           <para>How much did I spend on various expenses, grouped by month?</para>
         </question>
         <answer>
-          <para>As above</para>
-          <para>Sorting: Secondary Key Subtotal: Monthly</para>
+          <para>The following sequence will modify the above Transaction Report options
+          to subdivide each account into monthly groups and separately produce subtotals.</para>
+          <orderedlist>
+            <listitem><para>Select options as above</para></listitem>
+            <listitem><para><emphasis>Sorting</emphasis>: Secondary Key Subtotal: Monthly</para></listitem>
+          </orderedlist>
         </answer>
       </qandaentry>
       <qandaentry>
         <question>
           <para>How much did I spend on various expenses, with monthly comparison?</para>
         </question>
         <answer>
-          <para>    As above</para>
-          <para>General: Subtotal Table enabled</para>
+          <para>The following sequence will add a subtotal summary table in a grid, which
```

Comment:
"The following sequence will" repeated 3 times in succession is a bit stilted. Can you come up with a different lead in for each paragraph?